splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -316,3 +321,24 @@ def _stop_build_on_placeholder_usage(
                     exc_msg % ("input service", service["name"])
                 )
     global_config.update_schema_version("0.0.8")
+
+
+def _dump_enable_from_global_config(
+    global_config: global_config_lib.GlobalConfig,
+) -> None:
+    if global_config.has_inputs():
+        exc_msg = "`enable` attribute found in input's page table action."
+
+        # Fetch the table object from global_config
+        table = global_config.content.get("pages", {}).get("inputs", {}).get("table", {})
+
+        # Check if "enable" exists in the actions and remove it if present
+        actions = table.get("actions", [])
+        if "enable" in actions:
```

Comment:
when this function will be called? If we remove `enable` from the schema.json, it will fail anyway? Or does it run before schema validation? and the purpose is to streamline the upgrading?

Reply:
This function is called at the build time, and the script will automatically remove the `enable` from the globalConfig and update the schema version so that schema validation will not fail.